README.md

```diff
@@ -14,6 +14,8 @@ SPDX-License-Identifier: MIT
 
 The goal of the project is to build an open-source platform that empowers researchers to manage simple studies without needing deep technical expertise. This repository contains the frontend code for the platform. Visit [spezi-web-service-study-platform](https://github.com/StanfordSpezi/spezi-web-service-study-platform) for the backend code.
```

Comment:
While we are on it, it might be nice to add some more descriptive information here, maybe even several screenshots in tables similar to the other Spezi repos? Can help guide a user through the functionality 👍